apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -259,6 +265,20 @@ export default {
     this.loaded = true;
   },
 };
+
+function getPageHeadings(tableMetadata: ITableMetaData): string[] {
```

Comment:
I try to understand why you prefer to have same function twice, I mean, it is in the option API and now also a seperate function? Is it because the function name helps you? Then the original name should be changedf?

Reply:
I'ts because I prefer to have the options api as short as possible, so if a computed is more than a one-liner I prefer to call a function with a name that explains what's going on. this makes it easier to at a glance see what's going on for the computeds, instead of having to filter out a lot of code.  If I then want to see the implementation, I can always ctrl-click the function to jump to it.
I do think the name in the computed is correct, as 'pageHeadings' is the actual value when used elsewhere. Whereas getPageHeadings is a function that gets called to create this value, hence the name difference.

This changed wasn't really necessary, but I had half a mind of rewriting some it into setup style, however that was too much work for this file so I didn't 😄